__tests__/app.test.js

```diff
@@ -199,6 +199,87 @@ describe('GET /api/articles/:article_id/comments ERRORS', () => {
 }
 )
 
+describe('POST /api/articles/:article_id/comments', () => {
+test('Should add a new comment for a valid article_id ', () => {
+
+    const newComment = {
+        username: 'butter_bridge',
+        body: 'Test Comment'
+    }
+    return request(app)
+            .post('/api/articles/1/comments')
+            .send(newComment)
+            .expect(200)
+            .then(response => {
+                expect(response.body.comment).toBeTruthy();
+                expect(response.body.comment.author).toBe(newComment.username);
+                expect(response.body.comment.body).toBe(newComment.body);
+     })
+});
+
+
+})
+
+describe('POST /api/articles/:article_id/comments ERRORS', () => {
+    test('400: return error if missing username or body', () => {
+        const invalidComment = {body: 'Test comment'};
+    
+        return request(app)
+          .post('/api/articles/1/comments')
+          .send(invalidComment)
+          .expect(400)
+          .then(response => {
+            expect(response.body).toEqual({ message: 'Required key missing' });
+          });
+      });
+      test('400: return error if article_id is invalid input', () => {
+        const commentData = {
+            username: 'Yusha',
+            body: 'Test comment'
+          };
+    
+        return request(app)
+          .post('/api/articles/one/comments')
+          .send(commentData)
+          .expect(400)
+          .then(response => {
+            expect(response.body).toEqual({ message: 'Invalid input' });
+          });
+      });
```

Comment:
Please use a valid username and invalid article_id. Don't do 2 errors in one test 🙂